source/index.html.md

```diff
@@ -2003,3 +2003,157 @@ it includes any accumulated interest.
 | margin ratio | 1e4 | `MARGIN_PRECISION` |
 | asset/liability weight | 1e4 | `SPOT_WEIGHT_PRECISION` |
 | imf weight precision | 1e6 | `SPOT_IMF_PRECISION` |
+
+# Drift Vaults
+Drift Vaults are permissionless programs that let users deposit and withdraw tokens from a shared pool managed by a single delegate. The delegate can only place or cancel orders on behalf of the vault, with no access to user funds.
+
+A more indepth explanation of vaults can be found on the [wiki](https://github.com/drift-labs/drift-vaults/wiki).
+
+## Vault cli
+Anyone can setup their own vault with desired configuration. The cli utility to aid in the creation of vaults, can be found [here](https://github.com/drift-labs/drift-vaults/tree/master/ts/sdk).
```

Comment:
there' also a UI example https://github.com/drift-labs/vaults-ui-template which has a manager page to create vaults now too https://vaults-ui-template.vercel.app/vault-manager/create-vault